src/main/scala/io/iohk/ethereum/BootstrapDownload.scala

```diff
@@ -0,0 +1,135 @@
+package io.iohk.ethereum
+
+
+import java.io.{File, FileInputStream, FileOutputStream}
+import java.net.URL
+import java.nio.file._
+import java.security.{DigestInputStream, MessageDigest}
+import java.util.zip.ZipInputStream
+
+import io.iohk.ethereum.utils.Logger
+
+
+/**
+  * A facility to
+  * - check the download location for a minimum amount of free space
+  * - download a zip from a URL and generate MD5 checksum
+  * - check the checksum
+  * - clean files out of given location
+  * - unzip to a given location
+  */
+object BootstrapDownload extends Logger {
+
+  val bufferSize = 4 * 1024
+  val leveldbFolderName = "leveldb"
+
+  private def assertAndLog(cond: Boolean, msg: String): Unit = {
+    if(!cond) log.info(msg)
```

Comment:
What's this for? The message will be shown anyway, won't it?

Reply:
The exe on windows will not shoe console messages. So it goes in the log too.